src/pages/SamplePage.tsx

```diff
@@ -1,38 +1,11 @@
-import { Col, PageHeader, Row } from "antd";
 import React from "react";
-import SampleIndicatorCard from "../components/SampleIndicatorCard";
-
+import SamplePaper from "../components/SamplePaper";
 const SamplePage: React.FC = () => {
-    
-    return (
+  return (
     <React.Fragment>
```

Comment:
If there is only one Component, you don't need the `<React.Fragment>`. 
From a semantical point of view, I would put the `<Paper>` components always into the Pages, and use compontents at deeper entrypoints. I think this is the intention of `<Paper>`.